tests/unit/test_dt_webhook.py

```diff
@@ -0,0 +1,59 @@
+"""Unit tests for dt_webhook.py."""
+# pylint: disable-next=import-error
+import pytest
+from aiohttp import web
+
+from extensions.eda.plugins.event_source.dt_webhook import (
+    _parse_auth_header,
+    _set_app_attributes,
+)
+
+args = {
+    "host": "127.0.0.1",
+    "port": 1234,
+    "token": "thisisnotanactualtoken",
+}
+
+url = f'http://{args["host"]}:{args["port"]}/event'
```

Comment:
is `url` actually needed for unit test? Or did you miss a unit test where it should be used?

Reply:
No good catch. I will remove it.